cli/src/main/kotlin/com/wire/kalium/cli/commands/LoginCommand.kt

```diff
@@ -49,7 +49,9 @@ class LoginCommand : CliktCommand(name = "login") {
     }
 
     private suspend fun provideVersionedAuthenticationScope(serverLinks: ServerConfig.Links): AuthenticationScope =
-        when (val result = coreLogic.versionedAuthenticationScope(serverLinks).invoke()) {
+        when (val result = coreLogic.versionedAuthenticationScope(serverLinks).invoke(
```

Comment:
The previous comment will solve this as well.

Reply:
we can, but I don't know if it's good to default values in use cases !